networking/cidr-range-definitions.adoc

```diff
@@ -17,7 +17,7 @@ Machine CIDR ranges cannot be changed after creating your cluster.
 ====
 
 ifdef::openshift-rosa,openshift-dedicated[]
-When specifying subnet CIDR ranges, ensure that the subnet CIDR range is within the defined Machine CIDR. You must verify that the subnet CIDR ranges allow for enough IP addresses for all intended workloads, including at least eight IP addresses for possible AWS Load Balancers.
+When specifying subnet CIDR ranges, ensure that the subnet CIDR range is within the defined Machine CIDR. You must verify that the subnet CIDR ranges allow for enough IP addresses for all intended workloads for possible AWS Load Balancers or Google Load Balancers.
```

Comment:
Good point. I actually removed the reference to both AWS and GCP because when looking at it again, since this is an ifdef for both OSD and ROSA, it makes sense to have it be more generic rather than have GCP mentioned in the ROSA documentation.